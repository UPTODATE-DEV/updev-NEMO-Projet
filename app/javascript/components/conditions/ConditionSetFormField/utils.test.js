import { getLevelsValues, applyDefaultLevelsValues } from './utils';

it('gets levels values', () => {
  const result = getLevelsValues([
    { name: 'foo', selected: 'bar' },
    { name: 'baz', selected: 'qux' },
  ]);
  expect(result).toMatchSnapshot();
});

it('applies default levels values', () => {
  const result = applyDefaultLevelsValues([
    { name: 'foo', selected: 'bar' },
    { name: 'baz', selected: null },
  ], {
    baz: 'qux',
  });
  expect(result).toMatchSnapshot();
});
