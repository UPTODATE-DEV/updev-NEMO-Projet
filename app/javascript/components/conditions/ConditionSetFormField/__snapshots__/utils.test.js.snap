// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`applies default levels values 1`] = `
Array [
  Object {
    "name": "foo",
    "selected": "bar",
  },
  Object {
    "name": "baz",
    "selected": "qux",
  },
]
`;

exports[`gets levels values 1`] = `
Object {
  "baz": "qux",
  "foo": "bar",
}
`;
